packages/comms/src/ecl/dfuWorkunit.ts

```diff
@@ -154,6 +154,20 @@ export class DFUWorkunit extends StateObject<UDFUWorkunitState, IDFUWorkunitStat
         });
     }
 
+    update(request: Partial<UpdateRequest>): Promise<FileSpray.UpdateDFUWorkunitResponse> {
+        return this.connection.UpdateDFUWorkunitEx({
+            wu: {
+                ...request,
+                ID: this.ID,
+                State: this.State
+            },
+            ClusterOrig: this.ClusterName,
+            JobNameOrig: this.JobName,
+            isProtectedOrig: this.isProtected,
+            StateOrig: this.State
```

Comment:
Why is update needed?  For protected?  Any fields that should not be changed, you should be able to omit.

Reply:
The ECL Watch UI only allows JobName and Protected to be updated currently.